.gitignore

```diff
@@ -67,7 +67,6 @@ coverage
 dist
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30187&commitId=3c8bf66c992712c7a824452626a059136ecf02ca&env=prod&clientType=FLUENTUI)


  <details><summary><b>Chat</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Chat.Chat Example Header Override.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>